app/webpacker/controllers/talk-to-us_controller.js

```diff
@@ -0,0 +1,24 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+
+    static targets = ["chat", "launch", "tta"];
+
+    connect() {
+        this.showChatControls();
+    }
+
+    showChatControls() {
+        this.chatTarget.style.display = 'inline-block';
```

Comment:
Would it be nicer to have this in CSS and just toggle a class name?

Reply:
debatable one.  if we end up adding more attribute with the action then yes.
but I dont think thats the plan.
its not a good idea to have like ...

.hidden { display: none }
.block { display: block }

etc, as you are then micro-abstracting the css logic